### DISCUSSION TITLE
Segmented control

### DISCUSSION THREAD

**Discussion by Varshni11** [saadeghi/daisyui#3064] [Ideas / feature requests]
We can have segmented control component which is used widely in websites these days.

**Comment by saadeghi** [comment]
We don't call it as segmented control but it's visually the same thing as tabs:  
https://daisyui.com/components/tab/#boxed

You can also use radio inputs with `btn` class:  
https://daisyui.com/components/pagination/#using-radio-inputs

Let me know if you have any questions

**Reply by Varshni11** [reply]
Thanks for letting me know.
